README.md

```diff
@@ -176,6 +144,97 @@ GTO the artifact file is committed to Git.
 
 </details>
 
+### Unassigning a stage
+
+Sometimes you need to mark an artifact version no longer ready for a specific
+consumer, and maybe signal a downstream system about this. You can use
+`gto untag` for that:
+
+```console
+$ gto untag awesome-model --version v0.0.1 --stage prod
+Created git tag 'awesome-model#prod#2!'
+```
+
+<details summary="Some details and options">
+
+GTO creates a special Git tag in a standard format:
+`{artifact_name}#{stage}#{e}!`.
+
+Note, that later you can create this stage again, if you need to, by calling
+`$ gto tag` again.
+
+You also may want to delete the git tag instead of creating a new one. This is
+useful if you don't want to keep extra tags in you Git repo, don't need history
+and don't want to trigger a CI/CD or another downstream system. For that, you
+can use:
+
+```console
+$ gto untag awesome-model --version v0.0.1 --stage prod --delete
+Deleted git tag 'awesome-model#prod#1'
+To push the changes upstream, run:
+git push origin awesome-model#prod#1 --delete
+```
+
+</details>
+
+### Deregister a version
+
+Sometimes you need mark a specific artifact version as a no longer ready for
+usage. You could just delete a git tag, but if you want to preserve a history of
+the actions, you may find `gto untag` useful.
+
+```console
+$ gto untag awesome-model --version v0.0.1
+Created git tag 'awesome-model@v0.0.1!' that deregistered a version.
+```
+
+<details summary="Some details and options">
+
+If you want to deregister the version by deleting the Git tags itself, you could
+use
+
+```console
+$ gto untag awesome-model --version v0.0.1 --delete
+Deleted git tag 'awesome-model@v0.0.1' that registered a version.
+Deleted git tag 'awesome-model#prod#1' that assigned a stage.
+Deleted git tag 'awesome-model#prod#2!' that unassigned a stage.
+To push the changes upstream, run:
+git push origin awesome-model@v0.0.1 awesome-model#prod#1 awesome-model#prod#2! --delete
```

Comment:
Could you add a clarification as to why `awesome-model#prod#1` and `awesome-model#prod#2!` are also included?